packages/gatsby-plugin-aphrodite/.gitignore

```diff
@@ -1,2 +0,0 @@
-/*.js
-!index.js
```

Comment:
Out of curiosity, what error were you seeing here? This could just be an accidental regression on Babel's side.

Reply:
babel now exits with a non-zero if the cli call doesn't end up with any files that it can transpile. It's probably not a bad change, but it messes with our scaffolding a bit, which assumes you'll use babel, but these stub packages don't actually have any files in `src/`

Reply:
Gotcha. I don't know if that was an intentional change or not on our end. I feel like as long as the directory itself exists, the fact that there are no files shouldn't be an error.

Reply:
Filed: https://github.com/babel/babel/issues/7337